app/service/CasesService.scala

```diff
@@ -457,8 +460,11 @@ class CasesService @Inject() (
   def getAssignedCases(pagination: Pagination)(implicit hc: HeaderCarrier): Future[Paged[Case]] =
     connector.findAssignedCases(pagination)
 
-  def updateCase(caseToUpdate: Case)(implicit hc: HeaderCarrier): Future[Case] =
-    connector.updateCase(caseToUpdate)
+  def updateCase(caseToUpdate: Case, operator: Operator)(implicit hc: HeaderCarrier): Future[Case] =
+    for {
+      updatedCase <- connector.updateCase(caseToUpdate)
+      _ = auditService.auditCaseUpdated(caseToUpdate, updatedCase, operator)
```

Comment:
This does not include the original case, it will add the same case twice to the audit event I think?

I think we need to add an extra argument to this function that takes the case before it was updated.